data-raw/process_data_set_danica.R

```diff
@@ -0,0 +1,64 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+library(janitor)
+library(usethis)
+
+#Historical Raptor by Player
+historical_raptor_by_player <- read_csv("data-raw/nba-raptor/historical_RAPTOR_by_player.csv") %>%
+  mutate(
+    era = factor(c("historical"))
+  ) 
+historical_raptor_by_player[c("raptor_box_offense", "raptor_box_defense","raptor_box_total", 
+            "raptor_onoff_offense", "raptor_onoff_defense", "raptor_onoff_total")] <- NA
+historical_raptor_by_player <- historical_raptor_by_player[c("era","player_name","player_id","season","poss","mp","raptor_box_offense","raptor_box_defense","raptor_box_total",    
```

Comment:
Could you do this with a `select()`? Again, so that we have consistency of tidyverse style for this package and not base R?

Reply:
Make sure to put the previous `mutate()` and this `select()` all in the same pipe `%>%` chain.